Stratis.Bitcoin/Consensus/ConsensusValidator.cs

```diff
@@ -14,56 +14,89 @@
 
 namespace Stratis.Bitcoin.Consensus
 {
-	public class ConsensusValidator
+	/// <summary>
+	/// A set of options with default values of the Bitcoin network
+	/// This can be easily overridable for alternative networks (i.e Stratis)
+	/// Capital style param nameing is kept to mimic core
+	/// </summary>
+	public class ConsensusOptions
 	{
-		NBitcoin.Consensus _ConsensusParams;
-		const int MAX_BLOCK_WEIGHT = 4000000;
-		public const int WITNESS_SCALE_FACTOR = 4;
-		public const int SERIALIZE_TRANSACTION_NO_WITNESS = 0x40000000;
+		public int MAX_BLOCK_WEIGHT = 4000000;
```

Comment:
Why are these becoming writable fields?
If for overriding, consider:
`public virtual int MAX_BLOCK_WEIGHT {get {return 4000000;}}`